app/controllers/prizes_controller.rb

```diff
@@ -11,20 +11,30 @@ def create
     render json: prize
   end
 
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
   def win_prize
```

Comment:
Looking at this method, it seems that what we are doing is basically:
- sort a new prize
- assign the prize to the lead
- send an email to the lead

So, to fix the warnings about the method being too long and doing way too much complex things, I'd like you to refactor the `win_prize` method into a method that just calls three other methods. Something like:
```
def win_prize
  sort_prize
  assign_prize_to_lead
  send_email

  render ...
end
```

Reply:
I fixed the win_prize method, it's now calling the other methods, which are returning values that are being passed to the next methods. Is it a valid solution?

Reply:
Not only is it a valid solution, it's also more idiomatic Rails. 

It's very common to have classes in Rails that have the public methods be mostly calls to private methods. It helps keep code clear and concise and forces us to name chunks of code that otherwise might be tricky to understand their purpose.

It's easier for the reader to understand what the `win_prize` method does by reading the private methods names than trying to understand what the entire implementation does

Good job 👍